### PR TITLE
Feature/url module

### PR DESCRIPTION
**Pull request by julsemaan**
# Description

Adds support for URL based module to allow redirection to internal or external URLs in the dynamic routing
# Impacts

Captive portal
# Delete branch after merge

YES
# NEWS file entries
## Enhancements
- Added module to redirect to internal or external pages within the portal modules configuration

### CODE REVIEW

**Comment by julsemaan** [comment]
Please review

File: html/captive-portal/lib/captiveportal/DynamicRouting/Module/URL.pm [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+BEGIN { extends 'captiveportal::PacketFence::DynamicRouting::Module::URL'; }
+
+=head1 NAME
+
+captiveportal::DynamicRouting::Module::URL - URL Controller for captiveportal
+
+=head1 DESCRIPTION
+
+[enter your description here]
```
Comment (jrouzierinverse) [inline html/captive-portal/lib/captiveportal/DynamicRouting/Module/URL.pm:12]: Please do

File: html/pfappserver/lib/pfappserver/Form/Config/PortalModule/URL.pm [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+has_field 'url' =>
+  (
+   type => 'Text',
+   label => 'URL',
+   required => 1,
+   tags => { after_element => \&help,
+             help => 'The URL on which the user should be redirected.' },
+  );
+
+sub child_definition {
```
Comment (jrouzierinverse) [inline html/pfappserver/lib/pfappserver/Form/Config/PortalModule/URL.pm:46]: documentation

File: html/pfappserver/lib/pfappserver/Form/Config/PortalModule/URL.pm [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+    return qw(url skipable);
+}
+
+sub BUILD {
```
Comment (jrouzierinverse) [inline html/pfappserver/lib/pfappserver/Form/Config/PortalModule/URL.pm:56]: documentation

File: html/captive-portal/lib/captiveportal/DynamicRouting/Module/URL.pm (hunk 1 above)
Comment (julsemaan) [inline html/captive-portal/lib/captiveportal/DynamicRouting/Module/URL.pm:12]: That's the empty extension point. It should have that text

**Comment by julsemaan** [comment]
Re-review

**Pull request merged**